journalapp/scripts/get_crisewing_api.py

```diff
@@ -0,0 +1,81 @@
+"""Script to retrieve existing journal entries in Cris Ewing's database."""
+
+import os
+import sys
+import requests
+from journalapp.scripts import initializedb
+from datetime import datetime
+from journalapp.models import Entry, DBSession
+import transaction
+
+DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f'
+
+
+def get_api_key():
+    """Return API key from system global environmental variables."""
+    key = os.environ.get('CRISEWING_API_KEY')
+    if not key:
+        print('crisewing API key not found in os.environ.')
+        sys.exit()
+    return key
+
+
+def get_api_response(key):
+    """Return a Response object from the requests module."""
+    url = 'https://sea401d2.crisewing.com/api/export?apikey={}'.format(key)
+    return requests.get(url)
+
+
+def format_datetime(string):
+    """Return a datetime object from the proper format."""
+    return datetime.strptime(string, DATETIME_FORMAT)
```

Comment:
What happens if the string can't be parsed according to your `DATETIME_FORMAT`?  Does your calling code handle that?